packages/worker/src/sdk/users/users.ts

```diff
@@ -479,7 +479,7 @@ export const bulkDelete = async (
     (user: RowResponse<User>) => {
       return user.doc
     }
-  )
+  ) as User[]
```

Comment:
I think this can be removed, due to the const already having the type assigned. Tried locally and it builds ok